source/features/index.js

```diff
@@ -0,0 +1,181 @@
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+
+export const features = {
+	/* GENERAL */
+	keyboardShortcuts: {
+		id: 'feature-keyboard-shortcuts',
+		category: 'general',
+		label: 'Enable keyboard shortcut to toggle Night Mode (Alt + M)',
+		fn: require('./keyboard-shortcuts').default
+	},
+	preserveTextMessages: {
+		id: 'feature-preserve-text-messages',
+		category: 'general',
+		label: 'Preserve unsent text in the Messages modal when it closes',
+		fn: require('./preserve-text-messages').default
+	},
+	useSystemFont: {
+		id: 'feature-use-system-font',
+		category: 'general',
+		label: 'Use the system font',
+		init: true
+	},
+
+	/* HEADER */
+	addLikesButtonNavBar: {
+		id: 'feature-likes-button-navbar',
+		category: 'header',
+		label: 'Add a "Likes" button to the main navbar',
+		fn: require('./likes-button-navbar').default,
+		init: true
+	},
+	cleanNavbarDropdown: {
+		id: 'feature-clean-navbar-dropdown',
+		category: 'header',
+		label: 'Cleanup navbar dropdown',
+		fn: require('./clean-navbar-dropdown').default
+	},
+	cleanupSearchSuggestions: {
+		id: 'feature-cleanup-search-suggestions',
+		category: 'header',
+		label: 'Remove hashtags suggestions in the search popover',
```

Comment:
This should not be configurable.

Reply:
Why not

Reply:
See the "Highlights" in the readme:

- [Removes the annoying suggestions in the search popover.](https://user-images.githubusercontent.com/170270/33800304-70198358-dd3d-11e7-9870-477a44f74f4d.png)

The behavior is super annoying. It pushes down the other results while you try to select something. I can't imagine anyone wanting that. If we're gonna make this configurable, we should at least move the hashtags results to the bottom so they don't move the actually useful results.

Reply:
@sindresorhus  is right tt's super annoying, it's my top reason to be able to use this extension.